app/models/team-membership.ts

```diff
@@ -0,0 +1,14 @@
+import Model, { attr, belongsTo } from '@ember-data/model';
+import type TeamModel from './team';
+import type UserModel from './user';
+
+export default class TeamMembershipModel extends Model {
+  @attr('date') declare createdAt: Date;
```

Comment:
Consistency: Place relationships first, and then separate by a newline and add attributes after

<img width="839" alt="Screenshot 2024-02-29 at 12 37 35" src="https://github.com/codecrafters-io/frontend/assets/3893573/f988b2ba-8d1e-426d-86bc-8b226e9da081">